yellowbrick/datasets/download.py

```diff
@@ -113,8 +116,9 @@ def load_game(data_path=FIXTURES):
     """
     # name of the dataset
     name = 'game'
-    data = load_numpy(name, data_path=data_path)
-    return data
+    path = _lookup_path(name, data_path=data_path)
+    dtype = np.array(['S1']*42+['|S4'])
+    return np.genfromtxt(path, dtype=dtype, delimiter=',', names=True)
```

Comment:
Had to make these changes to get the documentation example to work. Changes still required in our load datasets methodology.

Reply:
yeah, i noticed that earlier that some of the dtypes still need to be defined properly